apps/ledger-live-mobile/src/screens/Manager/Benchmarking.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import styled from "styled-components/native";
+import { useAppInstallProgress } from "@ledgerhq/live-common/apps/react";
+import Config from "react-native-config";
+
+const Wrapper = styled(Flex).attrs({
+  bg: "palette.background.default",
+  p: 4,
+  mt: 4,
+  borderWidth: 1,
+  borderRadius: 4,
+  borderColor: "neutral.c40",
+})``;
+
+const Status = styled(Flex).attrs(p => ({
+  bg: p.active ? "success.c100" : "warning.c100",
+  borderRadius: 20,
+  borderWidth: 0,
+  height: 10,
+  width: 10,
+}))``;
+
+export default function Benchmarking(props: any) {
+  const enabled = Config.ENABLE_BENCHMARKING || false;
+  const [list, setList] = useState([]);
+  const [start, setStart] = useState(0);
```

Comment:
just TS, but it's complaining where the `setStart` is used that `start` is a number, not a `Date`. Maybe `useState<Date | null>(null)` or if you find a `Date.zero` constant you don't need the null.